scripts/zkai_attention_derived_d128_outer_proof_route_gate.py

```
@@ -0,0 +1,669 @@
+        "slice_count": EXPECTED_SLICES,
+        "edge_count": EXPECTED_EDGES,
+        "first_blocker": FIRST_BLOCKER,
+    }
+    if summary != expected_summary:
+        raise AttentionDerivedD128OuterProofRouteError("summary drift")
+    if payload.get("non_claims") != NON_CLAIMS:
+        raise AttentionDerivedD128OuterProofRouteError("non_claims drift")
+    if payload.get("validation_commands") != VALIDATION_COMMANDS:
+        raise AttentionDerivedD128OuterProofRouteError("validation command drift")
+    if ("cases" in payload) != ("case_count" in payload):
+        raise AttentionDerivedD128OuterProofRouteError("mutation finalization drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)